views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
I know, but the readers of this article are from all walks of life - not immediately doing sign elimination and showing them the direct deconstruction, and then in the next step, the rewrite to Berstein form, is far nicer to them than doing it in one step in the assumption they follow along no problem.